docs/spec.md

```diff
@@ -544,6 +544,18 @@ implementation-defined. Tensor constants have the following constraints:
     has_shape(tensor_literal..., shape[1:])`.
   * otherwise, `false`.
 
+```ebnf
+QuantizedTensorConstant ::= QuantizedTensorLiteral ':' QuantizedTensorType
```

Comment:
This non-terminal also needed to be added to the `Constant ::= ...` rule.

Reply:
done.